src/reboost/hpge/utils.py

```diff
@@ -31,6 +31,14 @@
         "last_global_evtid",
     ],
 )
+FileInfo.__doc__ = "NamedTuple storing the information on the input files"
```

Comment:
there should be a nicer way to have namedtuples with docs (i.e. by subclassing)

example: https://github.com/legend-exp/legend-pygeom-l200/blob/8ab9cdaae3a1d8ab0986beaebc704a5e91711795/src/l200geom/core.py#L17-L37